app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -251,6 +270,17 @@ private void navigateTo(Class<? extends GdgActivity> activityClass, Bundle addit
         mDrawerLayout.closeDrawers();
     }
 
+    protected void setNavDrawerItemSelected(String title) {
+        Menu menu = mNavigationView.getMenu();
```

Comment:
I think that could work. Them to handle the drawer item clicks we could handle the special events in the "default" case of the switch. Since they all would have different ID's.